resources/assets/js/components/global/Verify.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<p
+		v-if="showBadge"
+		:class="['verify', 'icon', verified && 'verify--verified', canVerify && 'verify--can_verify']"
+		@click="onClick"
+	>
+		<file-verified-svg class="verify__svg" />
+		<span>{{message}}</span>
+	</p>
+</template>
+
+<style lang="sass" rel="stylesheet/sass" scoped>
+	@import 'resources/assets/sass/variables'
+
+	.verify
+		font-size: $font-size-minus-1
+		font-weight: bold
+		color: $color-gray
+		text-transform: uppercase
+		width: auto
+
+		&--verified
+			color: $color-green
+
+		&--can_verify
```

Comment:
I believe this should be:
```suggestion
		&--can-verify
```